samza-log4j2/src/main/java/org/apache/samza/logging/log4j2/StreamAppender.java

```diff
@@ -214,7 +220,7 @@ public void append(LogEvent event) {
           metrics.bufferFillPct.set(Math.round(100f * logQueue.size() / DEFAULT_QUEUE_SIZE));
         }
       } catch (Exception e) {
-        System.err.println("[StreamAppender] Error sending log message:");
+        System.err.println(String.format("[%s] Error sending log message:", appenderName));
```

Comment:
Yes. it will be the same as what we write in log4j2.xml
e.g.
```
<Appender type="Stream" name="<whatismyname>">
      <Layout type="PatternLayout" pattern="xxx"/>
</Appender>
```
Then, the appenderName is <whatismyname>. This name should be unique in the log4j2.xml.

Reply:
I understand that log4j2.xml should have the Java class name of the appender.
my question was if we can drop the variable "appenderName" and get it from `getClass().getName` instead

Reply:
Seems appenderName is better to be used here because theoretically it is possible that multiple instances of StreamAppender (or its child appenders) are used at the same time. But appenderName is really a unique name for different appender instances which is set in log4j2.xml